src/plugins/template/backend/ops/multiply.cpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/reference/multiply.hpp"
+
+#include "evaluate_node.hpp"
+
+template <ov::element::Type_t ET>
+bool evaluate(const std::shared_ptr<ov::op::v1::Multiply>& op, ov::TensorVector& outputs, const ov::TensorVector& inputs) {
+    using T = typename ov::element_type_traits<ET>::value_type;
+    ov::reference::multiply<T>(inputs[0].data<T>(), inputs[1].data<T>(), outputs[0].data<T>(), ov::shape_size(inputs[0].get_shape()));
+    return true;
+}
+
+template <>
+bool evaluate_node<ov::op::v1::Multiply>(std::shared_ptr<ov::Node> node,
+                                         ov::TensorVector& outputs,
+                                         const ov::TensorVector& inputs) {
+    auto element_type = node->get_output_element_type(0);
+
+    switch (element_type) {
+    case ov::element::boolean:
+        return evaluate<ov::element::boolean>(ov::as_type_ptr<ov::op::v1::Multiply>(node), outputs, inputs);
+    case ov::element::bf16:
```

Comment:
According @praasz comment, could you please leave only types which are not supported in `evaluate` method of `Multiply`